CHANGELOG.md

```diff
@@ -5,6 +5,101 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] - October, 31, 2023
+
+### React
+
+#### Added
+
+- [Footer] Added onLogoClick callback
+- [Container] Added a new attribute to align it with Header
+- [Login] New login components and utilities to provide OIDC authentication and api tokens
+
+#### Changed
+
+- [Checkbox, DateInput, Dropdowns, FileInput, NumberInput, PasswordInput, PhoneInput, SelectionGroup, NumberInput, TextInput, TextArea, TimeInput] Improved readability and accessibility of error, success and info states
+
+#### Fixed
+
+- [StepByStep] Fixed React warnings about missing keys.
+- [Header] onDidChangeLanguage is now called only when language is changed.
+- [Header] Fixed header dropdown overflow issues
+- [DatePicker] Fixed selected date getting reseted in a controlled state
+- [CookieConsent, Dropdown, FileInput, NumberInput, SearchInput, TextInput] Fixed button focus inside input components
+- [Tag] Fixed margins with several lines long texts
+
+### Core
+
+#### Changed
+
+- [Checkbox, DateInput, Dropdowns, FileInput, NumberInput, PasswordInput, PhoneInput, SelectionGroup, NumberInput, TextInput, TextArea, TimeInput] Improved readability and accessibility of error, success and info states
```

Comment:
Remove DateInput, Dropdowns, FileInput, TimeInput because Core doesn't have those. For copy paste reasons here you go:
`- [Checkbox, NumberInput, PasswordInput, PhoneInput, SelectionGroup, NumberInput, TextInput, TextArea] Improved readability and accessibility of error, success and info states`

Reply:
Updated